packages/create-waku/src/index.ts

```diff
@@ -185,13 +199,13 @@ async function init() {
     process.exit(1);
   }
   let repoInfo: RepoInfo | undefined;
-  const example = getExample(tokens);
 
-  if (example) {
+  // FIXME this is pretty hard to follow. Separate functions?
```

Comment:
@ojj1123 I found the code pretty hard to read. There is a big room for improvement. I will factor out the functionality into the separate file, and you can consider some refactors there. Please work on refactors and adding tests before adding new capabilities.